modules/end-user-guide/pages/optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc

```diff
@@ -0,0 +1,111 @@
+:_content-type: CONCEPT
+:description: Optional parameters for the URLs for starting a new workspace
+:keywords: url, urls, parameter, parameters, starting, start, workspace, workspaces, url-parameter, url-parameters, multiple-url-parameters, combine-url-parameters, several-url-parameters, add-url-parameters, concatenate, concatenate-multiple-URL-parameters, concatenate-URL-parameters, concatenating, concatenating-multiple-URL-parameters, concatenating-URL-parameters, concatenation, start-duplicate-workspaces, start-duplicated-workspaces, duplicate-workspace, duplicate-workspaces, duplicated-workspace, duplicated-workspaces, how-to-start-a-duplicate, how-to-start-duplicate, devfile, override-devfile-file-name, override-file-name, file-name, devfile-name, different-devfile-name, file-path, devfile-path, devfile-file-path, devfile-path-parameter, devfile-file-path-parameterdevfile, file-path, devfile-path, devfile-file-path, devfile-path-parameter, devfile-file-path-parameter
+:navtitle: Optional parameters for the URLs for starting a new workspace
+// :page-aliases:
+
+[id="parameters-for-the-urls-for-starting-a-new-workspace_{context}"]
+= Optional parameters for the URLs for starting a new workspace
+
+For the URLs for starting a new workspace, the following optional parameters provide an extra layer of basic workspace configuration in addition to the devfile.
+//here link to a section about the devfile. max-cx
+
+== URL parameter concatenation
+
+The URL for starting a new workspace supports concatenation of multiple URL parameters by using `&` with the following URL syntax:
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?__<url_parameter_1>__&__<url_parameter_2>__&__<url_parameter_3>__
+----
+
+== URL parameter for the workspace IDE
+
+Che-Theia is currently the default integrated development environment (IDE) that a workspace loads with if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.
+
+The URL parameter for specifying another supported IDE is `che-editor=__<editor_key>__`.
+
+The URL for starting a new workspace has the following syntax with the `che-editor=__<editor_key>__` parameter:
+
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?che-editor=__<editor_key>__
+----
+
+.`__<editor_key>__` values for supported IDEs
+|===
+| IDE | `__<editor_key>__` value | Note
+
+ifeval::["{project-context}" == "che"]
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+| `eclipse/che-theia/latest`
+| This is the default IDE with link:https://github.com/che-incubator/chectl/[chectl stable]. This IDE loads in a new workspace without entering this URL parameter.
```

Comment:
```suggestion
| This is the default IDE that loads in a new link:https://github.com/che-incubator/chectl/[chectl stable] workspace.
```
If you think readers might be confused, you can further re-enforce the point by adding "You do not need to specify it by using a URL parameter."

Reply:
@rolfedh, I rewrote the content in this column as follows:
![image](https://user-images.githubusercontent.com/78564304/161771956-9863b3a9-fed3-4426-98ba-805309a84b67.png)